### DISCUSSION TITLE
set admin command

### DISCUSSION THREAD

**Discussion by samentec** [mehedijaman/laravel-zkteco#4] [Q&A] [answered]
hello how can i change a user level roll to admin roll ?

**Comment by mehedijaman** [comment, marked as answer]
Set Users
//    set user

//    1 s't parameter int $uid Unique ID (max 65535)
//    2 nd parameter int|string $userid ID in DB (same like $uid, max length = 9, only numbers - depends device setting)
//    3 rd parameter string $name (max length = 24)
//    4 th parameter int|string $password (max length = 8, only numbers - depends device setting)
//    5 th parameter int $role Default Util::LEVEL_USER
//    6 th parameter int $cardno Default 0 (max length = 10, only numbers

//    return bool|mixed

    $zk->setUser();

**Reply by samentec** [reply]
tanks 
now how can i send the name (3rd parameter) with utf8 i am using in  persian(فارسی) language and when i set user in my device the name showing like this Ø§Ø­ÙØ¯Û  how can i fix this ?

**Reply by mehedijaman** [reply]
Unfortunately, ZKTeco machines don't support utf8 fonts. It only supports English. So you have to input name in English.

**Reply by samentec** [reply]
But my device supported Persian lang and it's frameware is Persian too

**Reply by mehedijaman** [reply]
Which model is it ?

**Reply by samentec** [reply]
K14
